openstackgit-update.yml

```diff
@@ -23,7 +23,7 @@
 # is expected to contain more repositories than RPC will need, but
 # will always have a complete set.
 
-- name: Update the openstackgit clones
+- name: Fetch the latest openstackgit clones
   hosts: localhost
   connection: local
```

Comment:
The git repositories do not need an artifact version. They simply need the latest copy of the git repo. An up to date copy will always contain the SHA's required for previous series and all versions.